models.py

```diff
@@ -235,35 +235,93 @@ def update(self, data, current_user):
         self.put()
 
     @staticmethod
-    def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng, start_date, end_date,
-                           fatal, severe, light, inaccurate, show_markers=True, is_thin=False, yield_per=None):
+    def bounding_box_query(is_thin=False, yield_per=None, **kwargs):
+
         # example:
         # ne_lat=32.36292402647484&ne_lng=35.08873443603511&sw_lat=32.29257266524761&sw_lng=34.88445739746089
         # >>>  m = Marker.bounding_box_query(32.36, 35.088, 32.292, 34.884)
         # >>> m.count()
         # 250
 
-        if not show_markers:
+        if not kwargs.get('show_markers', True):
             return Marker.query.filter(sql.false())
-        accurate = not inaccurate
         markers = Marker.query \
-            .filter(Marker.longitude <= ne_lng) \
-            .filter(Marker.longitude >= sw_lng) \
-            .filter(Marker.latitude <= ne_lat) \
-            .filter(Marker.latitude >= sw_lat) \
-            .filter(Marker.created >= start_date) \
-            .filter(Marker.created < end_date) \
+            .filter(Marker.longitude <= kwargs['ne_lng']) \
+            .filter(Marker.longitude >= kwargs['sw_lng']) \
+            .filter(Marker.latitude <= kwargs['ne_lat']) \
+            .filter(Marker.latitude >= kwargs['sw_lat']) \
+            .filter(Marker.created >= kwargs['start_date']) \
+            .filter(Marker.created < kwargs['end_date']) \
             .order_by(desc(Marker.created))
         if yield_per:
             markers = markers.yield_per(yield_per)
-        if accurate:
+        if kwargs.get('accurate', True) and not kwargs.get('approx', True):
             markers = markers.filter(Marker.locationAccuracy == 1)
-        if not fatal:
+        elif kwargs.get('approx', True) and not kwargs.get('accurate', True):
```

Comment:
If you're getting an entry more than once (with a default value) then extract it to a variable.